<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stackedWidget</class>
 <widget class="QStackedWidget" name="stackedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trademax API Client</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="page">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../resources/logo.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>501</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_purchase_orders">
       <property name="styleSheet">
        <string notr="true">min-height: 1.5em;</string>
       </property>
       <property name="text">
        <string>Ink√∂psordrar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_about">
       <property name="styleSheet">
        <string notr="true">min-height: 1.5em;</string>
       </property>
       <property name="text">
        <string>Om applikationen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
